translate/src/core/editor/hooks/useHandleShortcuts.ts

```diff
@@ -138,7 +140,11 @@ export function useHandleShortcuts(): (
             setEditorFromHelpers(translation, sources, true);
           } else {
             const { translation } = otherLocaleTranslations[nextIdx];
-            setEditorFromHelpers(getSimplePreview(translation), [], true);
+            setEditorFromHelpers(
+              getPlainMessage(translation, entity.format),
+              [],
+              true,
+            );
```

Comment:
This was previously working only accidentally, as non-Fluent messages would fail to parse and therefore fall back to their string source.